service/grpc/codelingo/codelingo.pb.go

```diff
@@ -69,34 +69,6 @@ func (m *DescribeFactRequest) String() string            { return proto.CompactT
 func (*DescribeFactRequest) ProtoMessage()               {}
```

Comment:
exported method DescribeFactRequest.Descriptor should have comment or be unexported